apps/web/modules/ui/components/id-badge/index.test.tsx

```
@@ -0,0 +1,139 @@
+
+    const copyButton = container.querySelector('button[aria-label="Copy CLICK-TEST"]')!;
+
+    // Button should be clickable and not throw error
+    expect(() => fireEvent.click(copyButton)).not.toThrow();
+  });
+
+  test("prevents event propagation when copy button is clicked", () => {
+    const parentClickHandler = vi.fn();
+
+    const { container } = render(
+      <div onClick={parentClickHandler}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)